src/util/std_expr.h

```diff
@@ -121,6 +121,18 @@ class symbol_exprt : public nullary_exprt
   }
 };
 
+namespace std
```

Comment:
This is probably harmless but I think it’d nevertheless be preferable if we could find a way to do without this.